software/programming_instructions.md

```diff
@@ -17,7 +16,11 @@ If you already have any version of the firmware or any other code loaded onto yo
   
 5. Continue to [Setting Up](#setting-up) as normal.
 
-  
+ ## Optional Quick start
+
+ The quickest way to get your EuroPi flashed with the latest firmware is to head over to the [releases](https://github.com/Allen-Synthesis/EuroPi/releases) page and download the latest `europi-vX.Y.Z.uf2` file. Then follow the 'BOOTSEL' instructions above to flash the EuroPi firmware to your pico. You can now skip the **Setting Up** section and proceed to the [Next Steps](#next-steps) section.
+
+> **_NOTE:_**  The EuroPi firmware and contrib scripts will be installed and accessible via Thonny for writing custom scripts on the pico, however Thonny will not show those firmware and contrib files in the pico filesystem. If you want to have access to the EuroPi firmware source code on your pico, follow the **Setting Up** instructions in the next section.
```

Comment:
[question] Have we tested that this is correct? Does a `main.py` located in the standard location override the one delivered with the firmware? It makes sense that it does, but I'd like to be sure.

This opens up a bunch of other questions for me about how things will work, but I guess we'll need to document them as we find out exactly how people want to use this new distribution.

Reply:
I can double check, but this follows the [import priority](https://github.com/Allen-Synthesis/EuroPi/blob/main/software/create_custom_firmware_uf2.md#import-priority) documentation @redoxcode provided.

Reply:
The testing I did was done using REPL, I didn't test adding a `main.py` and rebooting, so I'll test that to be sure.

Reply:
@mjaskula Good catch! My assumption was incorrect. I have updated the docs to reflect that `main.py` will not override the entry point.

We essentially now have 3 firmware options. 1) uf2 quick start drag and drop, 2) pypi distributable source, and 3) download source from GitHub. 

I think the updates I've included here are good for now, but we should probably revisit the programming instructions at some point.